lib/cli/Cargo.toml

```diff
@@ -149,14 +149,17 @@ experimental-io-devices = [
 ]
 singlepass = [
     "wasmer-compiler-singlepass",
+    "wasmer/singlepass",
```

Comment:
This is not correct

Reply:
Please undo

Reply:
If I remove this, it will error with "engine is not compiled in" when using `Store::default()`. This is the only way to pass the "singlepass" feature to the `api` crate, which is necessary for code that uses `#[cfg(feature = "singlepass")]`.

Otherwise `Store::default()` will error out. Not sure what the problem is from a dependency standpoint, if we want singlepass to be enabled in the CLI, it should obviously also be enabled for code that uses `Store::default()` (the dependencies are the same).

Reply:
Undone in https://github.com/wasmerio/wasmer/pull/3483/commits/8d3e7c0dadc638cec97265415a46c32f035ae073